test/cppsuite/tests/hs_cleanup.cxx

```diff
@@ -62,8 +62,22 @@ class hs_cleanup : public test {
         while (tc->running()) {
             tc->sleep();
 
-            if (tc->next(cursor) != 0)
-                continue;
+            auto ret = cursor->next(cursor.get());
+            if (ret != 0) {
+                if (ret == WT_NOTFOUND)
+                    continue;
```

Comment:
If we ever get `WT_NOTFOUND`, since the cursor is not being reset, are we gonna ever get something else than `WT_NOTFOUND` ?

Reply:
f407e9b